definitions/npm/numeral_v2.x.x/flow_v0.25.x-/numeral_v2.x.x.js

```diff
@@ -69,3 +69,144 @@ declare var numeral: Numeral;
 declare module 'numeral' {
     declare module.exports: typeof numeral
 }
+
+
+declare module 'numeral/locales' {
+  declare module.exports: void
```

Comment:
Instead of `void` shouldn't this be `$PropertyType<$Exports<'numeral'>, 'locales'>`?

Reply:
I don't think so. 
Locale modules don't export anything, they just register their locale settings via `numeral.register()`. That's why I call them "side-effect only modules".
You can see it in [one of the locale modules](https://github.com/adamwdraper/Numeral-js/blob/master/src/locales/pt-br.js).

Reply:
All right, thanks.